website/docs/cloud-docs/projects/index.mdx

```diff
@@ -0,0 +1,48 @@
+---
+page_title: Projects - Terraform Cloud
+description: >-
+  Use projects to organize and group workspaces and create ownership boundaries
+  across your infrastructure. 
+---
+
+# Overview
+
+-> **Note:** Projects are available to all users, but managing project
```

Comment:
I think this should be Standard edition? Could we also tweak this to match the [current call-out](https://developer.hashicorp.com/terraform/cloud-docs/workspaces/organize-workspaces-with-projects) to keep the consistent formatting of pricing callouts?

Reply:
Good catch, we actually have a partial that you can use: https://github.com/hashicorp/terraform-docs-common/blob/main/website/docs/partials/tfc-package-callouts/project-permissions.mdx